### PR TITLE
fixed go-lint issue in maya/pkg/apis/openebs.io/v1alpha1/register.go

### PR DESCRIPTION
**Pull request by cartinez**
Signed-off-by: Francesco Cartier <francescocartier@gmail.com>

**What this PR does / why we need it**: fixes a go-lint issue regarding comments for some exported vars

**Which issue this PR fixes**: fixes openebs issue [#2085](https://github.com/openebs/openebs/issues/2085)

**Special notes for your reviewer**: There wasn't really much to say in the comments, the function exported are basic initializations of runtime methods (if I got it right).

### CODE REVIEW

File: pkg/apis/openebs.io/v1alpha1/register.go [hunk 1]
```diff
@@ -4,8 +4,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	"github.com/openebs/maya/pkg/apis/openebs.io"
```
Comment (aswathkk) [inline pkg/apis/openebs.io/v1alpha1/register.go]: You should not remove this. It is being used in line number 12

Comment (cartinez) [reply pkg/apis/openebs.io/v1alpha1/register.go]: Sorry, I didn't mean to remove it, did that by accident probably. Fixing it right away.

File: pkg/apis/openebs.io/v1alpha1/register.go [hunk 2]
```diff
@@ -17,10 +17,12 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder initialization
 	SchemeBuilder runtime.SchemeBuilder
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
+	// AddToScheme initialization
+	AddToScheme = localSchemeBuilder.AddToScheme
```
Comment (prateekpandey14) [inline pkg/apis/openebs.io/v1alpha1/register.go]: @cartinez can you change given comments to something like this 

```suggestion
	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
	SchemeBuilder = runtime.NewSchemeBuilder
	// AddToScheme is a global function that registers this API group & version to a scheme
	AddToScheme = SchemeBuilder.AddToScheme

```

Comment (prateekpandey14) [reply pkg/apis/openebs.io/v1alpha1/register.go]: @cartinez you have to revert the previous commit as `SchemeBuilder = runtime.NewSchemeBuilder` added 2 times

File: pkg/apis/openebs.io/v1alpha1/register.go [hunk 3]
```diff
@@ -17,10 +15,12 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
 	SchemeBuilder runtime.SchemeBuilder
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
+	// AddToScheme is a global function that registers this API group & version to a scheme
+	AddToScheme = SchemeBuilder.AddToScheme
```
Comment (prateekpandey14) [inline pkg/apis/openebs.io/v1alpha1/register.go]: This should be the change i suppose ...sorry for the confusion.
```
// SchemeBuilder is the scheme builder with scheme init functions to run for this API package	
SchemeBuilder      runtime.SchemeBuilder

// AddToScheme is a global function that registers this API group & version to a scheme	
localSchemeBuilder = &SchemeBuilder
AddToScheme   = localSchemeBuilder.AddToScheme

```

File: pkg/apis/openebs.io/v1alpha1/register.go (hunk 1 above)
Comment (prateekpandey14) [inline pkg/apis/openebs.io/v1alpha1/register.go]: Don't have to remove this ...

Comment (cartinez) [reply pkg/apis/openebs.io/v1alpha1/register.go]: Sorry, messed up with the reverts. Fixing it.

File: pkg/apis/openebs.io/v1alpha1/register.go [hunk 4]
```diff
@@ -17,10 +16,11 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
 	SchemeBuilder runtime.SchemeBuilder
-	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
+	// AddToScheme is a global function that registers this API group & version to a scheme
+	AddToScheme        = SchemeBuilder.AddToScheme
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
```
Comment (prateekpandey14) [inline pkg/apis/openebs.io/v1alpha1/register.go]: @cartinez i think you are getting confused :slightly_smiling_face: please do this change.... 
see the order 
```sh
// SchemeBuilder is the scheme builder with scheme init functions to run for this API package	
SchemeBuilder      runtime.SchemeBuilder

// AddToScheme is a global function that registers this API group & version to a scheme	
localSchemeBuilder = &SchemeBuilder
AddToScheme   = localSchemeBuilder.AddToScheme

```

Comment (cartinez) [reply pkg/apis/openebs.io/v1alpha1/register.go]: oh boy, I'm doing a mess, sorry

**Pull request merged**